app/models/miq_alert.rb

```diff
@@ -210,6 +210,14 @@ def add_status_post_evaluate(target, result, event)
     miq_alert_statuses << status
   end
 
+  def find_status(target, event, ems_ref)
+    if event.event_type == "datawarehouse_alert"
```

Comment:
@simon3z @gtanzillo I think I have a nice solution for this which also solves an [indexing question](https://github.com/ManageIQ/manageiq/pull/14295#discussion_r106375893)